<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intérprete de sueños</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="next_e_btn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Episodio Siguiente</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prev_e_btn">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>130</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Episodio Anterior</string>
    </property>
   </widget>
   <widget class="GraphicsView" name="plotBarSueno">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>20</y>
      <width>681</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="GraphicsLayoutWidget" name="plotConsumo">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>191</y>
      <width>681</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
